packages/node/src/index.ts

```diff
@@ -373,36 +373,60 @@ export const build: BuildV3 = async ({
   );
   debug(`Trace complete [${Date.now() - traceTime}ms]`);
 
-  const project = new Project();
-  const staticConfig = getConfig(project, entrypointPath);
-
+  let routes: BuildResultV3['routes'];
   let output: BuildResultV3['output'] | undefined;
+
   const handler = renameTStoJS(relative(baseDir, entrypointPath));
+  const outputName = config.zeroConfig
+    ? handler.substring(0, handler.length - 3)
+    : handler;
+
+  // Will output an `EdgeFunction` for when `config.middleware = true`
+  // (i.e. for root-level "middleware" file) or if source code contains:
+  // `export const config = { runtime: 'experimental-edge' }`
+  let isEdgeFunction = false;
+
+  // Add a catch-all `route` for Middleware
+  if (config.middleware === true) {
```

Comment:
Where does this config come from? Is this like a feature flag?

Reply:
Added by the zero-config build match.

Reply:
Oh yeah I missed that 👍